.travis.yml

```diff
@@ -1,7 +1,10 @@
 sudo: required
 
-services:
-  - docker
+language: python
+python:
+  - 3.5
 
 script:
-  - docker-compose run async --quiet
+  - sudo pip install -r requirements.txt
```

Comment:
We need to implement a new `docker-compose` task like `diff-sync`. We should not install this requirement on PR because this is pretty slow and we already have the docker image with all that working fine.

I'm ok to merge that in this way if you can open a issue talking about that: "Create a new service on docker-compose for diff-sync". Or something like that.

Reply:
Agreed

Reply:
I meant that: https://github.com/DestructHub/ProjectEuler/blob/8b2e23b354c47235d9d10d0003bb33bc983ecef2/docker-compose.yml#L3